docs/tutorials/modo_arrays.md

```diff
@@ -0,0 +1,156 @@
+# Handling data arrays with MODO
+
+Any count-like data, e.g protein abundances, RNA counts, metabolomic measurments, etc. can be stored as arrays in the `MODO`.
+The underlying [zarr](https://zarr.readthedocs.io/en/stable/index.html) supports array creation as well as an interface to NumPy arrays.
+
+
+## Load data
+
+(pandas)=
+### Using panda DataFrames
+
+Count-like data can usually be loaded into [pandas DataFrame](https://pandas.pydata.org/docs/reference/frame.html).
+To keep column names (__observations__) and row names (__variables__) both need to be stored in a separate numpy array first:
+
+::::{tab-set}
+
+:::{tab-item} python
+:sync: python
+```{code-block} python
+import pandas as pd
+import numpy as np
+
+# Example of RNA-seq count data
+
+rna_count = pd.read_csv('/path/to/rna/counts.csv', index_col="gene")
+rna_count
+# rna_count
+#             time1  time2  time3    ...
+# gene
+# Xkr4        1891   2410   2159     ...
+# Rp1         2      2      0        ...
+# ...         ...    ...    ...      ...
+# TrnP        334    202    218      ...
+
+obs = rna_count.columns.to_numpy()
+var = rna_count.index.to_numpy()
+rna_array = rna_count.to_numpy()
+
+obs
+# array(['time1', 'time2', 'time3', ...], dtype=object)
+
+var
+# array(['Xkr4', 'Rp1', ..., 'TrnP'], dtype=object)
+
+rna_array
+# array([[1891, 2410, 2159, ...],
+#        [   2,    2,    0, ...],
+#        ...,
+#        [ 334,  202,  218, ...]])
+
+```
+::::
+
+:::{warning}
+`to_numpy()` automatically removes row and column names from pandas DataFrames.
+It is important to store them separately, if they contain important information.
+:::
+
+:::{note}
+Skip this section, if you already have your data in a NumPy array.
+:::
+
+## Add array element to MODO
+
+Next, an element with the metadata describing the array can be added to the `MODO`:
+
+::::{tab-set}
+
+:::{tab-item} python
+:sync: python
+```{code-block} python
+from modo.api import MODO
+import modo_schema.datamodel as model
+
+# load modo - example at "data/ex"
+modo= MODO("data/ex")
+
+# Generate an Array element
+array_element = model.Array(id="rna1", name= "RNA raw counts", description = "RNA counts from multiple timepoints", has_sample="sample/sample1", data_format = "Zarr")
```

Comment:
**comment:** I get 
```
ValueError: data_path must be supplied
```
adding `data_path` with a meaningless value seems to fix it. 

**question**: Should we open a separate issue for this? (i.e. make data_path optional)

Reply:
alternatively, we could also add a note saying that we plan to add helper functions (an array api) to simplify these operations

Reply:
Yes, very good point. I missed that - what about doing all of this:
1. In this PR: Fix the docs with the "path to where we will store the array"
2. In this PR: Add the note to make clear this is going to be wrapped/simplified
3. Open an issue to simplify this and make the data path optional (see https://github.com/sdsc-ordes/modo-api/issues/38)

Reply:
Sounds good to me!